plugins/source/aws/resources/services/identitystore/users_fetch.go

```diff
@@ -0,0 +1,5 @@
+package identitystore
+
+func fetchIdentitystoreGroups() error {
```

Comment:
Assuming this needs to be implemented?

Reply:
Yes, not done yet. There is a question in my mind about best way to use the value returned by `ssoadmin` service (identity store id) in the `identitystore` service. I am thinking I will need to call that API twice, once in each service. Let me know if you have a better idea.

Reply:
I made it work by declaring the group memberships table as a Relation in the codegen recipe.